mirage/scenarios/collections.ts

```diff
@@ -133,71 +151,338 @@ export function collectionModerationScenario(server: Server, currentUser: ModelI
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(285),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.Accepted,
+        toState: CollectionSubmissionReviewStates.Removed,
+        actionTrigger: CollectionSubmissionActionTrigger.ModeratorRemove,
+        comment: 'Your last chance is over buddy.',
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(275),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.InProgress,
+        toState: CollectionSubmissionReviewStates.Pending,
+        actionTrigger: CollectionSubmissionActionTrigger.Submit,
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        comment: 'Have a heart, Batman! Namaste ...',
+        dateCreated: getMinusDate(265),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.Pending,
+        toState: CollectionSubmissionReviewStates.Rejected,
+        actionTrigger: CollectionSubmissionActionTrigger.Reject,
+        target: collectionSubmission,
+        comment: 'I have told you before I am not into hot yoga, Joker!',
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(255),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.InProgress,
+        toState: CollectionSubmissionReviewStates.Pending,
+        actionTrigger: CollectionSubmissionActionTrigger.Submit,
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        comment: 'Catch me if you can spam man',
+        dateCreated: getMinusDate(245),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.Pending,
+        toState: CollectionSubmissionReviewStates.Rejected,
+        actionTrigger: CollectionSubmissionActionTrigger.Reject,
+        target: collectionSubmission,
+        comment: 'You thought I would not notice this horrible project?',
+        creator: projectBuilderArgument.currentUser,
+        dateCreated: getMinusDate(235),
+    });
+
+    projectBuilderArgument.server.create('collection-submission-action', {
+        fromState: CollectionSubmissionReviewStates.InProgress,
+        toState: CollectionSubmissionReviewStates.Pending,
+        actionTrigger: CollectionSubmissionActionTrigger.Submit,
+        target: collectionSubmission,
+        creator: projectBuilderArgument.currentUser,
+        comment: 'Fine, I have made the chances you requested',
+        dateCreated: getMinusDate(265),
```

Comment:
These last few submission-actions seem out of order on the date-created. Line 430, it was `dateCreated: getMinusDate(235)` and this one is `dateCreated: getMinusDate(265)`, with the actions made after this one going down by 10 from 265.